blog/_posts/2024-05-14-core-extension.md

```diff
@@ -0,0 +1,92 @@
+
+#### Broader region availability
+Core Kubernetes extensions are available in all regions across Azure Public cloud, Azure Government and China regions. See the full list of regions at [Product availability by region](https://azure.microsoft.com/explore/global-infrastructure/products-by-region/table) (search "Azure Arc enabled Kubernetes" for core Kubernetes extensions and "Azure Kubernetes Service (AKS)" for AKS in the list).
+
+#### Native AKS experience
+##### Creating and deleting core extensions
+* **Unchanged experience for core extensions migrated from add-ons**: Some AKS add-ons, such as [Container Insights](https://learn.microsoft.com/azure/azure-monitor/containers/container-insights-overview), [Managed Prometheus](https://learn.microsoft.com/azure/azure-monitor/metrics/prometheus-metrics-overview), and [Application Monitoring](https://learn.microsoft.com/azure/azure-monitor/app/kubernetes-codeless), are transitioning to core extensions. Although the underlying technology shifts to extensions, the API and the user experience in Azure CLI, Azure Portal and ARM/Bicep/Terraform/other templates remain unchanged.
+
+    CLI commands:
+
+    ```bash
+    az aks create/update --resource-group <group> --name <cluster> --enable/disable-<add-on>
+    ```
+* **More native experience for core extensions graduated from [standard extensions](https://learn.microsoft.com/azure/aks/cluster-extensions#currently-available-extensions)**: Currently, we are working on upgrading Azure Backup, which is key functionality, to core extensions for better integrated experience in AKS. More core extensions will come in the future.
+
+    CLI commands:
+    * Create/delete a **standard** extension:
+    ```bash
+    az k8s-extension create/delete --extension-type <type> --resource-group <group> --cluster-name <name> --cluster-type <clusterType> --name <extension name>
+    ```
+    * Create/delete a **core** extension:
+    ```bash
+    az aks extension create/delete --extension-type <type> --resource-group <group> --cluster-name <name> --name <core extension name>
+    ```
+
+#### Safer version management
+Core extensions follow the same [update rules as add-ons](https://learn.microsoft.com/azure/aks/integrations#add-ons): patch versions can be upgraded within a Kubernetes minor version, while major/minor upgrades occur only with Kubernetes minor version updates to avoid introducing breaking changes within a Kubernetes minor version. This policy balances reliability with the speed of availability of new features and patches.
+
+#### Improved security, efficiency and reliability
+
+Extension-manager is moved from user node pool to AKS Control Plane, which improves extension security, efficiency, and reliability.
```

Comment:
```suggestion
Extension-manager is moved from the user node pool to the AKS Control Plane, which improves extension security, efficiency, and reliability.
```

Reply:
Did we already talk about the networking advantages of this move elsewhere? If not, worth spelling them out here?